hypertrace-gradle-publish-plugin/README.md

```diff
@@ -3,12 +3,13 @@
 [![CircleCI](https://circleci.com/gh/hypertrace/hypertrace-gradle-publish-plugin.svg?style=svg)](https://circleci.com/gh/hypertrace/hypertrace-gradle-publish-plugin)
```

Comment:
existing issue: this is no longer relevant. There's a GHA equivalent badge if you want to grab that, but if you want to remove it, that's fine too.

Reply:
updated with GHA badge.